trade-finance-manager-api/src/v1/api.js

```diff
@@ -820,13 +821,35 @@ const getPartyDbInfo = async ({ companyRegNo }) => {
       url: `${EXTERNAL_API_URL}/party-db/${encodeURIComponent(companyRegNo)}`,
       headers: headers.external,
     });
-    return response.data;
+    return process.env.AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED === 'true' ? { status: 200, data: response.data } : response.data;
   } catch (error) {
     console.error('Unable to get party DB info %o', error);
+    if (process.env.AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED === 'true') {
```

Comment:
`isFeatureFlagEnabled` would be ideal

Reply:
Sure, have done - though worth noting that every other status code in this file is just a raw number, might be worth a tech debt ticket to refactor the rest of it. Feels messy for me to do that here as it would add to the diff quite substantially